common/plugins/org.jboss.tools.common/src/org/jboss/tools/common/log/BaseUIPlugin.java

```diff
@@ -11,14 +11,21 @@
 package org.jboss.tools.common.log;
 
 import java.util.HashMap;
+import java.util.Map;
 
```

Comment:
The changes to this file seem like they should be undone. They're just changes to imports